src/user/views/user_views.py

```diff
@@ -272,7 +272,7 @@ def leaderboard(self, request):
                         'reputation_records__amount',
                         filter=Q(**time_filter) & ~Q(reputation_records__distribution_type__in=['REFERRAL', 'REWARD', 'REFERRAL_APPROVED'])
                     ), 0)
-                ).order_by(F('hub_rep').desc(nulls_last=True), '-reputation')
```

Comment:
Will this break leaderboard sorting on specific hubs?

Reply:
This case is for the overall leaderboard -- the hub-specific case is just above this one and is unchanged.